spec/openapi.yaml

```diff
@@ -38,5 +38,7 @@ tags:
       description: These are the times that have been approved and will be used in payroll.
     - name: Stats
       description: Location statistics to assist with accurate scheduling.
+    - name: Background Job Processor
```

Comment:
I'm not completely happy with the "Background Job Processor". What about "Bulk Importer"? I know this then potentially limits us to use it only for importing and we have already been asked about syncing staff...

I think it is the word "background", I don't feel like giving out that facet of information for some reason, it makes me uneasy.

Maybe "Mass Processor", or "Batch Processor", or "Bulk Processor"? Open to ideas.

Reply:
I am happy with any of the three you suggested.

Reply:
Batch is my preference